content/rs/references/cli-utilities/rlcheck/_index.md

```diff
@@ -11,20 +11,60 @@ aliases: ["/rs/references/cli-reference/rlcheck/",
 The `rlcheck` utility runs various health checks on a Redis Enterprise Software node and reports any discovered issues.
```

Comment:
If health checks are the tests below, maybe we should call them tests in this first sentence to make it clear? Or maybe we could say the "rlcheck utility checks the health of a node by running various tests..."